include/IECoreScene/MeshPrimitive.h

```diff
@@ -113,6 +130,13 @@ class IECORESCENE_API MeshPrimitive : public Primitive
 		mutable int m_minVerticesPerFace;
 		mutable int m_maxVerticesPerFace;
 
+		IECore::ConstIntVectorDataPtr m_cornerIds;
```

Comment:
Ideally we'd have a breaking change note in the comment of this commit as well as the description of the PR

Reply:
Could I persuade you to rebase that in Matti, rather than me update my PR and then you synchronize it over again? I'm thinking things might be clearer if I just closed my PR...

Reply:
Absolutely. Will include that in the next update.

Reply:
3b4ef98 should have it.